wsclient/src/main/java/org/hpccsystems/ws/client/utils/Utils.java

```diff
@@ -691,4 +700,30 @@ public static boolean currentOSisLinux()
           }
           throw new IllegalArgumentException(enumclass.getName() +".'" + strvalue + "' is not valid.");
       }
+      
+      public static EclInfo GetEcl(String content) {
```

Comment:
I was always told that static methods started with capitals, and instance-level methods started with lowercase. Does HPCC-JAPI use lower-case method names for static methods too?